internal/fuzzbinder/util.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package fuzzbinder
+
+// This file provides utility functions for FuzzBinder tool.
+
+import (
+	"fmt"
+	"time"
+)
+
+// ValidateDate validates that the fuzzing date chosen to generate the fuzzing
```

Comment:
What do you think of calling the func `ValidateFuzzingDate`---as it has functionality specific to fuzzing, i.e., the 15 days retention.

Reply:
I agree !